cernopendata/config.py

```diff
@@ -386,7 +386,7 @@
                 }
             }
         ),
-        'post_filters': dict(
+        'filters': dict(
```

Comment:
There is a problem with this approach: do an empty search, click on "Datasets", and while the facet numbres are well updated, it is no longer possible to select "Software" in addition to "Datasets", say, because the other facet categories that are on Datasets level -- i.e. Documentation, Environment, etc -- disappear and are no longer visible.

It is important to be able to select multiple values, e.g. years 2010, 2011, 2012 from the choice.

Can you please look and amend so that multiple-value selection will behave as in PROD?

Reply:
While investigating, there are two different components `Pagination` and `ResultsPerPage` which shows the pagination bar in the bottom and X result per page respectively. The issue requires to access one prop of `ResultsPerPage` component i.e `currentSize` just outside the Pagination. Tried with state hook and passing of currentQueryState but no success so far.